src/parser.rs

```diff
@@ -112,12 +115,13 @@ pub fn parse(pairs: Pairs<Rule>) -> Result<Items> {
                         .collect()
                 }),
                 Rule::operator | Rule::number => Item::Raw(pair.as_str().to_owned()),
-                _ => (Item::TODO(pair.as_str().to_owned())),
+                _ => (Item::Todo(pair.as_str().to_owned())),
             })
         })
         .collect()
 }
 
+#[cfg(test)]
```

Comment:
How come these are marked for tests? Is it because they're not reachable from the CLI? And so we remove them as we make things reachable from there?

Reply:
Ya, right on! I'm sure there are a million ways to hush the lint warning, but this seemed reasonable. The compile will force you to remove these configuration conditions once an entrypoint uses these functions. I think it will help tease out a reasonable API.